docs/user_guide/secure_payloads.rst

```diff
@@ -0,0 +1,100 @@
+Secure Payloads
+================
+
+.. warning::
+    This page is still under development and not complete. It will be so until
+    this warning is removed.
+
+Secure payloads offer the ability to securely provision secrets to an enrolled node.
+
+The payload itself is sent via the Keylime Tenant to the verifier and only when a node
```

Comment:
I think this still needs correction -- the verifier never receives the payload.  The payload is always sent directly from the tenant to the node/agent.  The verifier's job is to send its key half (the `v_key`) to the node when the node passed its enrollment criteria. 

Maybe word this to say

> The encrypted payload itself is sent via the Keylime Tenant to the target node's Agent, along with the Tenant's `u_key` half.  Only when a node has passed its enrollment criteria (including any `tpm_policy` or IMA whitelist) will the Verifier send its `v_key` half to the Agent so that it can decrypt the payload.